client/go/src/utils/print.go

```diff
@@ -5,6 +5,7 @@
 package utils
```

Comment:
Nit: Package names should be singular.

Also, as I explained in https://github.com/vespa-engine/vespa/pull/18354#discussion_r692470370 - `src/` subdir is not necessary.

Reply:
Done, but it's not very nice ...